include/gatekeeper_gt.h

```diff
@@ -23,18 +23,60 @@
 
 #include <rte_atomic.h>
 
+#include "gatekeeper_lpm.h"
+#include "gatekeeper_config.h"
+
+/* Structure for Lua to set up the LPM routes. */
+struct lua_ip_routes {
+	const char *ip_addr;
+	uint8_t    prefix_len;
+	uint8_t    policy_id;
+};
+
+struct gt_match_fields {
+	union {
+		uint32_t v4;
+		uint8_t  v6[16];
+	} ip;
+	uint16_t proto;
+	uint16_t dest_port;
+};
+
 /* Structures for each GT instance. */
 struct gt_instance {
 	/* RX queue on the front interface. */
 	uint16_t      rx_queue;
 
 	/* TX queue on the front interface. */
 	uint16_t      tx_queue;
+
+	/* The lua state that belongs to the instance. */
+	lua_State     *lua_state;
+
+	/* The gt policy table. */
+	struct lpm_rt *lpm;
```

Comment:
Why is the LPM table held here instead of being created, edited, released in a Lua policy? Consider the case in which a policy wants to keep two or mroe LPM tables. Or, yet, a simple policy that doesn't need a LPM table at all.